google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/deprecated/Message.java

```diff
@@ -75,7 +75,7 @@ public PubsubMessage apply(Message message) {
     }
 
     @Override
-    protected ByteString getByteString() {
+    public ByteString getByteString() {
```

Comment:
Is it related to the RpcFuture -> ApiFuture rename? I is not related to the change, but it seems strange that the class Message under the "deprecated" directory is not declared as @Deprecated .